### PR TITLE
fix(rosetta): 'extract' does not translate samples in submodule READMEs

### PR DESCRIPTION
**Pull request by rix0rrr**
---

By submitting this pull request, I confirm that my contribution is made under the terms of the [Apache 2.0 license].

[Apache 2.0 license]: https://www.apache.org/licenses/LICENSE-2.0

### CODE REVIEW

**Review by RomainMuller — APPROVED** [review]

**Comment by rix0rrr** [comment]
I don't understand why the integ test is failing

**Comment by RomainMuller** [comment]
There is an *unhandled* promise rejection in `jsii-rosetta` (it exists successfully, but really failed):

```
UnhandledPromiseRejectionWarning: Error: Sample uses fixture with-lambda-trigger, but not found: /home/runner/work/jsii/jsii/aws-cdk/packages/aws-cdk-lib/rosetta/with-lambda-trigger.ts-fixture
```

It appears some CDK documentation block uses a non-existent fixture file!

**Comment by rix0rrr** [comment]
Thanks for digging that one out for me!

Guess there's no getting around the fact that the strategies Rosetta uses to get at fixtures and types aren't going to work for a monocdk that has been stitched together from the individual source packages.

Here's what Rosetta sees:

* Here we are extracting all the samples from `.../packages/aws-cdk-lib`.
* Some submodule has a README that uses a fixture named `with-lambda-trigger`.

Here's what Rosetta needs to conclude:

* Oooooh OF COURSE I know that file lives in `.../packages/aws-cdk-lib/lib/aws-lambda/rosetta/with-lambda-trigger.ts`.

....but how? I don't think it can.

Bottom line: this can't ever work without a lot more investment.

**Comment by rix0rrr** [comment]
Supposedly we can move all fixtures to the right location... but they'll still have the wrong import names.
